protocol/git/README.md

```diff
@@ -47,8 +47,9 @@ Write a [good commit message]. Example format:
 
     http://project.management-system.com/ticket/123
 
-If you've created more than one commit, use a rebase to squash them into
-cohesive commits with good messages:
+If you've created more than one commit, use a
+[rebase to squash](https://davidwalsh.name/squash-commits-git)
```

Comment:
Additionally, what about the [official GitHub docs](https://help.github.com/articles/about-git-rebase/)?